logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -0,0 +1,536 @@
+
+
+[id="elasticsearch-node-disk-high-watermark-reached"]
+== Elasticsearch Node Disk High Watermark Reached
+
+Elasticsearch attempts to relocate shards away from a node link:https://www.elastic.co/guide/en/elasticsearch/reference/6.8/disk-allocator.html[that has reached the high watermark].
+
+.Troubleshooting
+
+. Identify the node on which Elasticsearch is deployed.
++
+[source,terminal]
+----
+oc -n openshift-logging get po -o wide
+----
+
+. Check the disk space on each node.
++
+[source,terminal]
+----
+for pod in `oc -n openshift-logging get po -l component=elasticsearch -o jsonpath='{.items[*].metadata.name}'`; do echo $pod; oc -n openshift-logging exec -c elasticsearch $pod -- df -h /elasticsearch/persistent; done
+----
+
+. Check if the cluster is rebalancing.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/health?pretty | grep relocating_shards
+----
++
+If the command output shows relocating shards, the High Watermark has been exceeded. The default value of the High Watermark is 90%.
++
+The shards relocate to a node with low disk usage that has not crossed any watermark threshold limits.
+
+. To allocate shards to a particular node, free up some space.
+
+. Try to increase the disk space on all nodes.
+
+. If increasing the disk space is not possible, try adding a new data node to the cluster.
+
+. If adding a new data node is problematic, decrease the total cluster redundancy policy.
+
+.. Check the current `redundancyPolicy`.
++
+[source,terminal]
+----
+oc -n openshift-logging get es elasticsearch -o jsonpath='{.spec.redundancyPolicy}'
+----
++
+NOTE: If you are using a `ClusterLogging` CR, enter:
++
+[source,terminal]
+----
+oc -n openshift-logging get cl -o jsonpath='{.items[*].spec.logStore.elasticsearch.redundancyPolicy}'
+----
+
+.. If the cluster `redundancyPolicy` is higher than `SingleRedundancy`, set it to `SingleRedundancy` and save this change.
+
+. If the preceding steps do not fix the issue, delete the old indices.
+
+.. Check the status of all indices on Elasticsearch.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- indices
+----
+
+.. Identify an old index that can be deleted.
+
+.. Delete the index.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name> -X DELETE
+----
+
+
+.Additional resources
+
+* "redundancyPolicy: SingleRedundancy" in the "Sample `ClusterLogging` custom resource (CR)" in xref:../../logging/config/cluster-logging-configuring-cr.adoc#cluster-logging-configuring-crd_cluster-logging-configuring-cr[About the Cluster Logging custom resource]
```

Comment:
So, you want me to go to the link, then go to "sample clusterlogging" then navigate to the section "redundancyPolicy"? I'm not used to this much navigation in "additional resources." I'm used to going to one page that I'm linked to. Granted, your users might expect more navigation on their part. 

Also, the xref's link works, however, when I click the "cluster-logging-about-crd" I get a 404 error page:
<img width="1100" alt="Screen Shot 2021-06-11 at 11 36 31 AM" src="https://user-images.githubusercontent.com/70717303/121712053-46244d00-caa9-11eb-9750-e552521ff3d3.png">